scripts/metavar.py

```diff
@@ -294,6 +302,7 @@ def __init__(self, prop_dict, source, run_env, context=None,
             if 'units' not in prop_dict:
                 prop_dict['units'] = ""
             # end if
+            # DH* Why is the DDT type copied into the kind attribute?
```

Comment:
I would like to know the answer to this question. I don't think it's needed. I had to add a workaround in suite_objects.py for this, and I also found one in ccpp_prebuild (metadata_parser.py).

Reply:
Let's check if we really need this

Reply:
If I take it out, I get:
```
Invalid metadata variable property, 'ddt_type', at /Users/heinzell/work/ccpp-framework/ccpp-framework-debug-20231012/test/advection_test/test_host_mod.meta:49
```
So for the purpose of this PR I am not going to touch `ddt_type`